### PR TITLE
Update direct_link_generator.py

### PR DESCRIPTION
**Pull request by kamileecher**

### CODE REVIEW

File: bot/helper/mirror_utils/download_utils/direct_link_generator.py [hunk 1]
```diff
@@ -469,3 +523,28 @@ def gdtot(url: str) -> str:
...
+    And https://github.com/Jusidama18"""
+    return url.replace("dropbox.com", "dl.dropboxusercontent.com")
+
+
+def dropbox2(url: str) -> str:
+    """ Dropbox Downloader Folder """
+    return url.replace("?dl=0", "?dl=1")
+
+
+def useragent():
```
Comment (anasty17) [inline bot/helper/mirror_utils/download_utils/direct_link_generator.py:538]: How this works ??

File: bot/helper/mirror_utils/download_utils/direct_link_generator.py [hunk 2]
```diff
@@ -90,10 +94,18 @@ def direct_link_generator(link: str):
         return streamtape(link)
     elif 'bayfiles.com' in link:
         return anonfiles(link)
+    elif 'https://sourceforge.net' in link:
+        return sourceforge(link)
+    elif 'https://master.dl.sourceforge.net' in link:
```
Comment (anasty17) [inline bot/helper/mirror_utils/download_utils/direct_link_generator.py:99]: Remove https://

File: bot/helper/mirror_utils/download_utils/direct_link_generator.py [hunk 3]
```diff
@@ -90,10 +94,18 @@ def direct_link_generator(link: str):
         return streamtape(link)
     elif 'bayfiles.com' in link:
         return anonfiles(link)
+    elif 'https://sourceforge.net' in link:
```
Comment (anasty17) [inline bot/helper/mirror_utils/download_utils/direct_link_generator.py:97]: Remove https://

**Comment by alok99-git** [comment]
Btw original code from this repo https://github.com/Ncode2014/re-cerminbot/blob/master/bot/helper/mirror_utils/download_utils/direct_link_generator.py

**Pull request closed without merging**